Khan/wonder-blocks
# Fix mobile click delay in clickable-behavior

Test Plan:
using mobile emulation:
- tap the dropdowns, see that they are *much* more responsive (no 300ms
  delay)
- tap the checkboxes, it should also respond quickly!
- same with radios
- and modal buttons
